ndcube/utils/wcs.py

```diff
@@ -138,77 +145,120 @@ def _wcs_slicer(wcs, missing_axis, item):
 
     missing_axis: `list` of `bool`
         Altered missing axis list.  Note the ordering has been reversed to reflect the data
-        (numpy) axis ordering convention.
+        (numpy) axis ordering convention. 
+        
+    dropped_coords:
+        Coordinates which have been dropped in the slicing process is collected in a tuple called
+        `dropped_coords`. 
 
     """
+    # Check ordering. 
```

Comment:
Woops!  I forgot that the ```item``` can be an int or slice, not always a ```tuple```.  So this needs to be a bit more complex.
```python
# Force item to be WCS order if entered by user in numpy order.
if numpy_order is True:
    # Convert item to a tuple is not already so we can handle it in a common way.
    if isinstance(item, (int, np.int64, slice)):
        item = (item)
    # If item does not contain entries for all data axes, add empty slices for those axes.
    n_add_axes = sum(np.invert(missing_axis)) - len(item)
    item = tuple(list(item) + [slice(None)]*n_add_axes)
    # Reverse order of item to be in WCS order.
    item = item[::-1]
```

Reply:
Think I will need to master this part by heart...